src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -60,9 +60,14 @@ const ChartsHolder = (props: {
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
 
-  const { pathname } = useLocation();
+  const { pathname, hash } = useLocation();
   const isRecommendationsShareUrl = pathname.includes('recommendations');
 
+  const defaultMetric =
+    hash === '#explore-chart'
```

Comment:
Should this be "#explore-hospitalizations" maybe?  Else add a comment about how this hash is used to make this code clearer?

And rename "defaultMetric" to "defaultExploreMetric" ?